src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -40,23 +43,27 @@ const resolveRunDataWithPin = (runData, pinnedRun) => {
  * @param {object} props.trackingData The experiment tracking run data.
  */
 const RunDataset = ({
+  enableComparisonView,
   enableShowChanges,
   isSingleRun,
   pinnedRun,
   setRunDatasetToShow,
   setShowRunPlotsModal,
   trackingData,
 }) => {
+  if (!trackingData) {
+    return null;
+  }
+
   return (
-    <div
-      className={classnames('details-dataset', {
-        'details-dataset--single': isSingleRun,
-      })}
```

Comment:
You might still need this `details-dataset--single` class name. I'm seeing that the single-run view on your branch is closer together than we we had before (yours is the top, demo is the bottom):

<img width="874" alt="Screen Shot 2022-08-15 at 11 24 04" src="https://user-images.githubusercontent.com/16869061/184619879-072f830f-a981-477d-accb-c458f94d5f4f.png">

Reply:
ah yeah good point, I've used the "comparison view" classnames instead the "single-view". Should be updated now